docker/prod/prod.template.yaml

```diff
@@ -277,7 +277,7 @@ spec:
               cpu: 5
```

Comment:
I think the limit should be at most 2 x request

Reply:
Yeah, we just discussed it offline and I think we should leave cpu request as is for now, but reduce limit slightly.